javnov.Selenium.Axe/javnov.Selenium.Axe/AxeBuilder.cs

```diff
@@ -1,18 +1,29 @@
 ﻿using javnov.Selenium.Axe.Properties;
+using Newtonsoft.Json.Linq;
 using OpenQA.Selenium;
 using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Net;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace javnov.Selenium.Axe
 {
+
+    /**
+	 * Chainable builder for invoking aXe. Instantiate a new Builder and configure testing with the include(),
+	 * exclude(), and options() methods before calling analyze() to run.
+	 */
     public class AxeBuilder
     {
         private readonly IWebDriver _webDriver;
         private readonly string _scriptContent;
+        private IList<string> _includes;
```

Comment:
We dont need `IList`here. Also we can make it readonly `private readonly List<string> _includes = new List<string>()`